app/src/main/java/org/simple/clinic/bloodsugar/entry/BloodSugarEntryEffectHandler.kt

```diff
@@ -67,6 +67,7 @@ class BloodSugarEntryEffectHandler @AssistedInject constructor(
         .addTransformer(CreateNewBloodSugarEntry::class.java, createNewBloodSugarEntryTransformer())
         .addTransformer(FetchBloodSugarMeasurement::class.java, fetchBloodSugarMeasurement(schedulersProvider.io()))
         .addConsumer(SetBloodSugarReading::class.java, { ui.setBloodSugarReading(it.bloodSugarReading) }, schedulersProvider.ui())
+        .addAction(UpdateBloodSugarEntry::class.java, {})
```

Comment:
If we don't handle a effect that is being dispatched from the update function, effect handler would throw a error when that effect is received. To prevent that I have used a no-op condition

Reply:
I can see that you are implementing the operation in the next commit. 
Instead of this being empty which means that this `Effect` has no action which is not true, can we add `TODO`s wherever the implementation is in progress?

Reply:
Done